vowpalwabbit/parse_example_json.h

```diff
@@ -1670,11 +1670,11 @@ inline void apply_pdrop(vw& all, float pdrop, v_array<example*>& examples)
 {
   if (all.example_parser->lbl_parser.label_type == label_type_t::cb)
   {
-    for (auto& e : examples) { e->l.cb.weight = 1 - pdrop; }
+    for (auto& e : examples) { e->l.cb.weight /= 1 - pdrop; }
```

Comment:
[Minor] Consider adding a comment explaining what is going on here, mathematically.

Reply:
done